Archive/Imo/Imo2008Q2.lean

```diff
@@ -53,7 +53,7 @@ theorem imo2008_q2a (x y z : ℝ) (h : x * y * z = 1) (hx : x ≠ 1) (hy : y ≠
   have hn_ne_zero : n ≠ 0 := by contrapose! hx; field_simp; assumption
   have hmn_ne_zero : m + n ≠ 0 := by contrapose! hz; field_simp; linarith
   have hc_sub_sub : c - (c - m - n) = m + n := by abel
-  rw [ge_iff_le, ← sub_nonneg]
+  rw [← sub_nonneg]
```

Comment:
This one is actually necessary, because the theorem statement incorrectly uses ge

Reply:
thanks, added it back